Automattic/woocommerce-payments
# Branding rollout: Change WordPress.org assets for WCPay

Fixes #6669 #6339

#### Changes proposed in this Pull Request

This PR includes the following changes to be shown on the plugin page  https://wordpress.org/plugins/woocommerce-payments/:
- icons, banners, screenshots
- readme.txt

Also, it includes the following changes:
- Change Woo Payments occurrences to WooPayments as described in #6339 
- Small logo fix

#### Testing instructions

* Check out `update/6669-change-wordpress-assets-to-match-new-branding`
* Onboard new PO account. You'll have to remove the existing one and clear the cache if you have one.
* Check the PO eligibility modal. The text should be WooPayments instead of Woo Payments.
   ![image](https://github.com/Automattic/woocommerce-payments/assets/79862886/582fafe7-9a92-432c-9a72-832a5c8e8cb1)
* Check the WooPayments logo on the settings page. It should have equal padding from the top, left, and bottom.
   ![image](https://github.com/Automattic/woocommerce-payments/assets/79862886/6d1f0ce1-115e-495e-8403-e49849483ce5)
* New images in `wordpress-org-assets` should match the design
* Changes in readme.txt and other text occurrences should make sense and contain new brand name (WooPayments)

-------------------


**Post merge**
